backends/blocked/ceed-blocked-operator.c

```diff
@@ -381,12 +381,18 @@ static int CeedOperatorApply_Blocked(CeedOperator op, CeedVector invec,
   }
 
   // Zero lvecs
-  for (CeedInt i=0; i<numoutputfields; i++) {
-    ierr = CeedOperatorFieldGetVector(opoutputfields[i], &vec); CeedChk(ierr);
-    if (vec == CEED_VECTOR_ACTIVE)
-      vec = outvec;
-    ierr = CeedVectorSetValue(vec, 0.0); CeedChk(ierr);
-  }
+    for (CeedInt i=0; i<numoutputfields; i++) {
+      ierr = CeedOperatorFieldGetVector(opoutputfields[i], &vec); CeedChk(ierr);
+      if (vec == CEED_VECTOR_ACTIVE) {
```

Comment:
I set the convention that only the active output gets added to - the assumption being that passive outputs would be linked to the specific sub-operators.